client/student/App.vue

```diff
@@ -22,13 +22,17 @@ html, body {
   height: 100%;
   margin: 0;
   padding: 0;
-  background-color: #e5e5e5;
+  overflow: hidden;
 }
 
 #app {
+  display: flex;
   height: 100%;
+  padding: 32px 0;
+  flex-direction: column;
```

Comment:
Do we need this within a root (flex / flex-column). Seams like it can be moved to the main container.